Open Judge System/Web/OJS.Web/Areas/Contests/Views/Submissions/View.cshtml

```diff
@@ -3,11 +3,15 @@
 @using OJS.Web.Areas.Contests.Controllers
 
 @using Resource = Resources.Areas.Contests.Views.SubmissionsView
+@using AdministrationResource = Resources.Areas.Administration.AdministrationGeneral
 
 @model OJS.Web.Areas.Contests.ViewModels.Submissions.SubmissionDetailsViewModel
 
 @{
-    ViewBag.Title = string.Format(Resource.Title, Model.Id, Model.UserName, Model.ProblemName);
+    var action = Model.IsContestActive ? CompeteController.CompeteActionName : CompeteController.PracticeActionName;
+    var problemNameAsLink = $@"<a href=""/Contests/{action}/{GlobalConstants.Index}/{Model.ContestId}#{Model.ProblemIndexInContest}"">{Model.ProblemName}</a>";
```

Comment:
Why not use `Html.ActionLink` here instead of manually typing out the URL?

Reply:
I have to append `#{index of the problem in the TabStrip}` in the end of the url, that's why I cannot use Html.ActionLink, but instead I fixed it with Url.Action(), which is also much better than doing it manually